{ 'test_4': { 'left_arm': [ { 'duration': 1.91,
                              'name': u'high_step_7',
                              'positions': [ -0.17,
                                             -1.3223,
                                             -0.11,
                                             -0.22,
                                             0.06,
                                             -0.02,
                                             0.19],
                              'starttime': 0.0}],
              'left_leg': [],
              'neck': [],
              'right_arm': [ { 'duration': 1.91,
                               'name': u'high_step_7',
                               'positions': [ 0.17,
                                              1.3223,
                                              0.11,
                                              0.22,
                                              -0.06,
                                              0.02,
                                              -0.19],
                               'starttime': 0.0}],
              'right_leg': [],
              'torso': [ { 'duration': 1.91,
                           'name': u'high_step_7',
                           'positions': [0.0, 0.754],
                           'starttime': 0.0}]}}
